core/src/main/java/com/megaease/easeagent/core/Bootstrap.java

```diff
@@ -77,9 +77,7 @@
     public static void start(String args, Instrumentation inst, Iterable<Class<?>> providers,
                              Iterable<Class<? extends Transformation>> transformations) throws Exception {
         long begin = System.nanoTime();
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("Injected class: {}", AppendBootstrapClassLoaderSearch.by(inst, ClassInjector.UsingInstrumentation.Target.BOOTSTRAP));
-        }
+        LOGGER.info("Injected class: {}", AppendBootstrapClassLoaderSearch.by(inst, ClassInjector.UsingInstrumentation.Target.BOOTSTRAP));
```

Comment:
When I review codes, I generally thought codes of printing logs are unimportant and ignored them,  so I strongly recommend putting the loading statement out of `LOGGER.info` block.

Reply:
fixed